src/basic/list.h

```diff
@@ -139,14 +139,17 @@
 /* The type of the iterator 'i' is automatically determined by the type of 'head', and declared in the
  * loop. Hence, do not declare the same variable in the outer scope. Sometimes, we set 'head' through
  * hashmap_get(). In that case, you need to explicitly cast the result. */
+#define LIST_FOREACH_WITH_NEXT(name,i,n,head)                           \
```

Comment:
Shouldn't this be `_LIST_FOREACH_WITH_NEXT()` to match `_LIST_FOREACH_WITH_PREV()` ?

Reply:
The macro `LIST_FOREACH_WITH_NEXT()` is explicitly used in udev-rules.c.